### PR TITLE
Feature java doc

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding java doc to book model and controller
## Trello Card
https://trello.com/c/jGPH4hsk

### CODE REVIEW

File: src/main/java/com/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -31,44 +31,72 @@
...
+     */
     public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
             Model model) {
         model.addAttribute(NAME, name);
         return GREETING_VIEW;
     }
 
     @PostMapping
+    /**
+     * @param bookVO the book with the attributes that can choose the user to create it
```
Comment (torsello) [inline src/main/java/com/wolox/training/controllers/BookController.java]: Wherever an object is declared on java doc, I recommend you to declare it like this: {@link BookVO}, so when you generate the documentation, you will see a link to this Object :) 

Replace it in every case.

**Review by torsello — CHANGES_REQUESTED** [review]

File: src/main/java/com/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/controllers/BookController.java]: Good tip Mati, I pushed the change!

**Review by torsello — APPROVED** [review]
Cool!

**Review by jdhiguita — APPROVED** [review]
Great job!

**Pull request merged**